pkg/controller/operators/olm/labeler.go

```diff
@@ -6,7 +6,6 @@ import (
 
 	"github.com/operator-framework/api/pkg/operators/v1alpha1"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry/resolver/cache"
-	"github.com/operator-framework/operator-registry/pkg/registry"
```

Comment:
`non-blocking question`: Would it make sense to keep the default name and remove the alternatively named one (opregistry) instead?

Reply:
Answering both comments: totally fair question - I mainly went with my gut locally and tried to avoid removing the package import alias that was used more frequently. The one caveat is more common/standardized package imports that are used and I value consistency over standardization generally speaking.